README.md

```diff
@@ -1 +1,104 @@
-# SpatialConnect
+
+#### Fresh install of EFC and the associated dependencies.
+
+#### Dependencies List
+What you will need to have installed prior to running the program:
+
+##### Git repositories
+* [spatialconnect-mobile](https://github.com/boundlessgeo/spatialconnect-mobile)
+`
+git clone https://github.com/boundlessgeo/spatialconnect-mobile
+`
+
+* [react-native-maps](https://github.com/boundlessgeo/react-native-maps)
```

Comment:
currently we are using the airbnb npm version of react-native-maps, so this is not needed for now